web/modules/custom/joinup_user/joinup_user.module

```diff
@@ -342,8 +488,24 @@ function joinup_user_og_membership_presave(EntityInterface $entity) {
 }
 
 /**
- * Implements hook_ENTITY_TYPE_predelete() for OG Membership entities.
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Disallow access to the cancellation method.
  */
-function joinup_user_og_membership_predelete(EntityInterface $entity) {
-  joinup_user_og_membership_presave($entity);
+function joinup_user_form_user_multiple_cancel_confirm_alter(&$form, FormStateInterface $form_state, $form_id) {
+  joinup_user_form_user_cancel_form_alter($form, $form_state, $form_id);
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Disallow access to the cancellation method.
+ */
+function joinup_user_form_user_cancel_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
The user_cancel_delete_form is not the admin form. It's the delete confirmation page.
Why is this needed:
  Unlike other entities, the user entity does not have a 'edit own/any', 'delete/own any', 'select cancel method' etc. There is one permission to allow delete any user and this permission is 'administer users'. The same permission gives access to the cancellation method and the ability to choose whether to notify the user etc.